test/functional/feature_fedpeg.py

```diff
@@ -317,6 +317,13 @@ def run_test(self):
         fin_psbt = sidechain.finalizepsbt(signed_psbt['psbt'])
         assert_equal(fin_psbt, signed_pegin)
 
+        # Try funding a psbt with the peg-in
```

Comment:
Assert that the balance is 50 BTC and the outputs summed to too much for the value you're adding next line.

Reply:
Done